src/hooks/useIsSafeApp.ts

```diff
@@ -1,14 +1,11 @@
-import { useState } from 'react'
-
 import { useIsomorphicLayoutEffect } from '@/hooks/useIsomorphicLayoutEffect'
 
-export const useIsSafeApp = (): boolean => {
-  const [isSafeApp, setIsSafeApp] = useState(true)
+let isSafeApp = false
 
+export const useIsSafeApp = (): boolean => {
```

Comment:
I am in general quite unhappy with this hook.
I think a far more reliable variant would be to use the safe-apps-sdk for this.

I tried this and it worked quite well on a short test:
```ts

export const useIsSafeApp = (): boolean => {
  const { connected } = useSafeAppsSDK()
  return connected
}
```